tests/main/snapctl/task.yaml

```diff
@@ -14,6 +14,22 @@ execute: |
         exit 1
     fi
 
+    echo "Verify that snapctl -h and --help run for regular users"
+    for arg in "-h" "--help"; do
+        if ! su -c "snapctl $arg" test ; then
```

Comment:
Not a complaint about your work in any way but the order of arguments in `su` is just textbook example of terrible design.

Thank you for testing as a non-root user.

Reply:
Yes... it is terrible indeed.